app/containers/ViewQuestion/saga.js

```diff
@@ -167,6 +169,27 @@ export function* getQuestionData({
   yield put(getQuestionBountySuccess(bounty));
   question.isGeneral = isGeneralQuestion(question.properties);
 
+  if (promote && promote.ends_time > Math.trunc(Date.now() / 1000)) {
+    question.promote = promote;
```

Comment:
In this way you set link on object not object data. It would be better to use another syntax `question.promote = { ...promote };`